pkg/admission/validator.go

```diff
@@ -28,7 +28,7 @@ func (gs *ServerHandler) serve(w http.ResponseWriter, r *http.Request) {
 	var arResponse admissionv1.AdmissionReview
 	var body []byte
 	if r.Body != nil {
-		if data, err := ioutil.ReadAll(r.Body); err == nil {
+		if data, err := io.ReadAll(r.Body); err == nil {
```

Comment:
@baum apologies, this isn't related to this PR just caught my attention - I think we are missing `defer r.body.Close()` somewhere here.

Reply:
@utkarsh-pro thanks for the review.🖖. 

```
The Server will close the request body.
The ServeHTTP  Handler does not need to.
```
See [lines 176-177 in net/http/request.go](https://go.dev/src/net/http/request.go)

That is why I think we're OK here. @kfir-payne WDYT?

Reply:
I think that @baum is right here and we're OK.

Reply:
Ah, I didn't know that. Now I got the rethink every `defer req.Body.Close()` that I have written 😆 . Thank you for pointing to the source @baum 🙌 .